scylla-cql/src/types/serialize/row.rs

```diff
@@ -0,0 +1,47 @@
+use std::sync::Arc;
+
+use crate::frame::response::result::ColumnSpec;
+use crate::frame::value::ValueList;
+
+use super::SerializationError;
+
+pub struct RowSerializationContext<'a> {
+    columns: &'a [ColumnSpec],
+}
+
+impl<'a> RowSerializationContext<'a> {
+    pub fn columns(&self) -> &'a [ColumnSpec] {
```

Comment:
Nit: this method and `column_by_name` could be made `#[inline]` - they are small and simple, and without it the compiler won't consider inlining them in when used in external crates.

Reply:
Done